network/p2p/capabilities.go

```diff
@@ -109,17 +112,22 @@ func (c *CapabilitiesDiscovery) PeersForCapability(capability Capability, n int)
 	return peers, nil
 }
 
+const capAdvertisementInitialDelay = time.Second / 10000
+
 // AdvertiseCapabilities periodically runs the Advertiser interface on the DHT
 // If a capability fails to advertise we will retry every 100 seconds until full success
 // This gets rerun every at the minimum ttl or the maxAdvertisementInterval.
 func (c *CapabilitiesDiscovery) AdvertiseCapabilities(capabilities ...Capability) {
 	c.wg.Add(1)
 	go func() {
 		// Run the initial Advertisement immediately
-		nextExecution := time.After(time.Second / 10000)
+		nextExecution := time.After(capAdvertisementInitialDelay)
```

Comment:
Feels like this should not be a named constant, we're just trying to trigger immediate execution, so why not the lowest possible integer?
```suggestion
		nextExecution := time.After(1)
```
(And then the use in the test file is really about waiting for a log message, so it's not really 3 times this number, it's just some value that has to do with how long logging takes.)